libs/components/src/cascader/vl-cascader-node.component.ts

```diff
@@ -0,0 +1,73 @@
+import { CSSResult, html, PropertyDeclarations, TemplateResult } from 'lit';
+import { BaseLitElement } from '@domg-wc/common-utilities';
+import { customElement } from 'lit/decorators.js';
+import { resetStyle } from '@domg/govflanders-style/common';
+import { vlElementsStyle } from '@domg-wc/elements';
+import cascaderNodeUigStyle from './vl-cascader-node.uig-css';
+import { CASCADER_SLOTS, NodeData } from './vl-cascader.model';
+import { getNodesForSlot } from './vl-cascader.utils';
+import { VlCascaderComponent } from './vl-cascader.component';
+
+@customElement('vl-cascader-node')
+export class VlCascaderNodeComponent extends BaseLitElement {
+    item: NodeData = { label: 'label missing' };
+    label = 'label missing';
+    templateType: string | undefined;
+    requestParams?: unknown;
+    cascaderRef: VlCascaderComponent | undefined;
+    static get properties(): PropertyDeclarations {
+        return {
+            label: { type: String },
+            templateType: { type: String, attribute: 'template-type' },
+            item: { type: Object },
+        };
+    }
+
+    static get styles(): (CSSResult | CSSResult[])[] {
+        return [resetStyle, vlElementsStyle, cascaderNodeUigStyle];
+    }
+
+    constructor() {
+        super();
+        this.label = 'label missing';
+    }
```

Comment:
Deze default value zet je al vanboven in de file

Reply:
verwijderd